src/Nethermind/Nethermind.Blockchain/Synchronization/ISyncConfig.cs

```diff
@@ -33,7 +33,8 @@ public interface ISyncConfig : IConfig
         [ConfigItem(Description = "If set to 'true' then the Fast Sync (eth/63) synchronization algorithm will be used.", DefaultValue = "false")]
         bool FastSync { get; set; }
         
-        [ConfigItem(Description = "Relevant only if 'FastSync' is 'true'. If set to a value, then it will set a minimum height threshold limit up to which FullSync, if already on, will stay on when chain will be behind network. If this limit will be exceeded, it will switch back to FastSync. Please note that last 32 blocks will always be processed in FullSync, so setting it to less or equal to 32 will have no effect.", DefaultValue = "1024")]
+        // Minimum is taken from MultiSyncModeSelector.StickyStateNodesDelta
+        [ConfigItem(Description = "Relevant only if 'FastSync' is 'true'. If set to a value, then it will set a minimum height threshold limit up to which FullSync, if already on, will stay on when chain will be behind network. If this limit will be exceeded, it will switch back to FastSync. Please note that last 2 blocks will always be processed in FullSync, so setting it to less than 2 will have no effect.", DefaultValue = "1024")]
```

Comment:
Setting it to 2 is risky - better 32

Reply:
Added recommendation. It can be set to as low as 2, lets keep the docs accurate.